src/app/employee-privacy-policy/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { PageHeader } from '@/components/PageHeader'
+import { StructuredDataScript } from '@/components/StructuredDataScript'
+
+import { createMetadata } from '@/utils/createMetadata'
+
+import {
+  attributes,
+  react as Content,
+} from '@/content/pages/employee-privacy-policy.md'
+
+import { PATHS } from '@/constants/paths'
+
+import { generateStructuredData } from './utils/generateStructuredData'
+
+const { header, seo } = attributes
+
+export const metadata = createMetadata({ seo, path: PATHS.PRIVACY_POLICY.path })
+
+export default function EmployeePrivacyPolicy() {
+  const { title } = header
+
+  return (
+    <article>
+      <StructuredDataScript structuredData={generateStructuredData(seo)} />
+      <PageHeader.Title>{title}</PageHeader.Title>
```

Comment:
Should we include header here? As we are doing it with [SEO PR](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/174/files)?

```
<header>
    <PageHeader.Title>{title}</PageHeader.Title>
</header>
```